addons/website_sale/views/snippets/snippets.xml

```diff
@@ -88,6 +88,13 @@
                          data-customize-website-views="website_sale.products_description"
                          data-no-preview="true"
                          data-reload="/"/>
+            <we-checkbox string="Collapsed sidebar"
+                         id="collapsed_sidebar"
+                         data-customize-website-views="website_sale.products_attributes_collapsed, website_sale.products_categories_list_collapsed"
+                         data-no-preview="true"
+                         data-reload="/"
+                         data-dependencies="attributes_opt, categories_opt"
+            />
```

Comment:
Debatable... but citing a wise man:

> [...] because users will enable this option only if they have a large amount of items / filters

... and in this scenario chances are that you want always make both collapsible. I'd suggest to follow the specs for now. We can alway adapt the behaviour later on if the use-case proves to be valid 😉